<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectObjWindow</class>
 <widget class="QWidget" name="selectObjWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update</string>
  </property>
  <widget class="QLabel" name="mainHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select One</string>
   </property>
  </widget>
  <widget class="QComboBox" name="objInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
